microsoft-edge/devtools/performance/share-performance-traces.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Share a performance trace with more data
+description: Export and import performance traces in DevTools to resolve slow performance or excess memory usage issues in your webpage.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 11/06/2025
+---
+# Share a performance trace with more data
+
+The data that's recorded by the **Performance** tool can be exported to files on disk, which can also include annotations, resource content (HTML, CSS, and JavaScript content), and source maps.  The exported files are called _traces_.  An exported trace is a `.json` file that can be imported in DevTools at any time.
+
+When you export a trace from the Performance tool, you can choose whether to include additional data:
+* Annotations.
+* Resource content (such as JavaScript content).
+* Script source maps.
+
+You can also control whether the trace file is compressed.
+
+The **Performance** tool records runtime data about your webpage.  Exploring the recorded data makes it possible to improve your webpage's runtime performance.
+
+Exporting performance traces is useful when you want to share these files with other people, to get help with investigating issues.
+
+Traces can be saved to include annotations, resource content (such as scripts), and source maps from the webpage.  This additional information makes it easier to analyze an imported trace file, by recreating the environment in which the trace was recorded, and by providing original source files.
+
+Exported trace files are compatible with other browsers that are based on the Chromium engine.
+
+When a trace that includes annotations, resource content (such as scripts) or source maps is imported in DevTools, a new DevTools window appears.  This new window isn't connected to the webpage that's running in your browser, and instead re-creates part of the environment in which the trace was originally recorded.  This DevTools instance contains only the **Performance** and **Sources** tools.
+
+
+<!-- ====================================================================== -->
+## Export a trace from the Performance tool
+
+To record performance for aspects of a webpage, and then export a performance recording:
+
+1. Open a webpage, such as [To do](https://microsoftedge.github.io/Demos/demo-to-do/), in a new window or tab.
+
+1. Right-click the webpage, and then select **Inspect**.
+
+   DevTools opens.
+
+1. In DevTools, open the **Performance** (![Performance icon](./share-performance-traces-images/performance-icon.png)) tool.
+
+   <!-- Leah TODO: get new png of dialog to show annotations and resource content ![Microsoft Edge with DevTools, showing the Performance tool, with the Save trace button](./share-performance-traces-images/save-performance-trace-dialog.png) -->
```

Comment:
create `save-performance-trace-dialog.png` w/ **Include annotations" and "Resource content" instead of "Script content" checkbox